lib/Builder/ParseQueueProcessor.php

```diff
@@ -68,7 +67,10 @@ public function process(ParseQueue $parseQueue): void
     private function processFile(string $file): void
     {
         if (getenv('SHELL_VERBOSITY') >= 1) {
-            fwrite(STDERR, sprintf("Processing file: %s\n", $file));
+            $stdErr = fopen('php://stderr', 'wb');
+            if ($stdErr !== false) {
+                fwrite($stdErr, sprintf("Processing file: %s\n", $file));
```

Comment:
You are getting this issue because you are in the HTTP context, right? In the case of HTTP, where is this output?

Reply:
Yes, in the HTTP context. I don't know what happens with those messages. In that context I don't care much because I get the result as a value object with the list of error messages, etc.

In the CLI context I care about them, but I can get it easily.

Thanks!

Reply:
Ok, so if the issue is that we are in the HTTP context, how about checking `PHP_SAPI`?

Reply:
Yes, your solution looks better! I've updated the PR. Thanks.